django/contrib/admin/utils.py

```diff
@@ -489,12 +489,21 @@ def construct_change_message(form, formsets, add):
     Translations are deactivated so that strings are stored untranslated.
     Translation happens later on LogEntry access.
     """
+    # Evaluating `form.changed_data` prior to disabling translations is required
+    # to avoid fields affected by localization from being included incorrectly,
+    # e.g. where date formats differ such as MM/DD/YYYY vs DD/MM/YYYY.
+    changed_data = form.changed_data
+    with translation_override(None):
+        # Deactivate translations while fetching verbose_name for form
+        # field labels and using `field_name`, if verbose_name is not provided.
+        # Translations would happen later on LogEntry access.
+        changed_field_labels = _get_changed_field_labels_from_form(form, changed_data)
```

Comment:
Use form.changed_data directly and remove the alias as recommended by Tim.

Reply:
No, this was already discussed that `form.changed_data` needed to be built prior to disabling the translations. Read the comments added above. Tim's comment is outdated but the line it is on hasn't changed so GitHub hasn't hidden it.